app/models/user/permissions.rb

```diff
@@ -60,11 +60,11 @@ def member_permission_ids=(roles)
   end
 
   def member_permission_service_ids=(service_ids)
-    if service_ids.present?
-      service_ids = Array(service_ids).compact.map(&:to_i)
+    if service_ids.is_a? Array
```

Comment:
API and UI send these values in a "funny way", like `"[""]"` and `[""]`, because of the URL-encoded params limitations.

I changed the way the `service_ids` value is treated, so that it is less confusing to update the param directly with an `update` call.

So `member_permission_service_ids: []` sets "no services are allowed", whild `member_permission_service_ids: nil` sets to "all services are allowed".

Reply:
What if `service_ids` is neither and Array nor blank?
In the original code it was blank/non-blank but now there is a third possibility. Even if we return 4xx response, that would make sense.

Reply:
also it is a little strange to accept non-integer ids but lets say it was acceptable previously so we don't mak ea big change